Wireframe/index.html

```diff
@@ -1,13 +1,63 @@
+
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    
+
+    <img class="git-logo" src="photos/logo@2x.png" alt="">
+     <p id="index-p"> Git is the most commonly used version control system</p>
```

Comment:
I wonder why do you need id here? If it is used for styling, generally there is something else we use for styling. ids are unique and used most of the time for testing, JavaScript scripts or to override styling, which is not a case for you, I guess.